packages/components/psammead-most-read/src/item.jsx

```diff
@@ -1,60 +1,68 @@
-import React, { Fragment } from 'react';
-import { node, oneOf, shape, string } from 'prop-types';
+import React from 'react';
+import { node, shape, string } from 'prop-types';
 import styled from 'styled-components';
 import { getFoolscap, getDoublePica } from '@bbc/gel-foundations/typography';
+import { GEL_GROUP_3_SCREEN_WIDTH_MIN } from '@bbc/gel-foundations/breakpoints';
 import { C_EBON, C_POSTBOX } from '@bbc/psammead-styles/colours';
 import { scriptPropType } from '@bbc/gel-foundations/prop-types';
-import { GEL_SPACING } from '@bbc/gel-foundations/spacings';
 import {
   getSerifLight,
   getSerifMedium,
 } from '@bbc/psammead-styles/font-styles';
 
-export const StyledCountSpan = styled.span`
+export const MostReadRank = styled.span`
   ${({ script }) => script && getFoolscap(script)};
   ${({ service }) => getSerifLight(service)}
   color: ${C_POSTBOX};
   margin: 0; /* Reset */
-  padding-bottom: ${GEL_SPACING};
+  padding: 0;
   display: inline-block;
-  min-width: 3rem;
+  width: 3rem;
 `;
 
 const StyledLink = styled.a`
   ${({ script }) => script && getDoublePica(script)};
   ${({ service }) => getSerifMedium(service)}
   color: ${C_EBON};
+  
   text-decoration: none;
-  padding-bottom: ${GEL_SPACING};
   
   &:hover,
   &:focus {
     text-decoration: underline;
   }
 `;
 
-export const MostReadItem = ({
-  dir,
+const StyledItem = styled.div`
+  padding-top: 0.3rem;
+  padding-left: 0.3rem;
+
+  @media (min-width: ${GEL_GROUP_3_SCREEN_WIDTH_MIN}) {
+    padding-top: 0.5rem;
+    padding-left: 1rem;
```

Comment:
`padding-left` needs to be flipped to `padding-right` when the direction of the text is RTL. 

To to this, we can have this helper at the top of the file.

```
const paddingStart = ({ dir }) => `padding-${dir === 'ltr' ? 'left' : 'right'}`;
```
Then we can use it on this line
```
${paddingStart}: ${GEL_SPACING_DBL};
```

Reply:
Resolved, thanks for the fix!